src/Files.App/UserControls/SideBar/SideBarControls.xaml

```diff
@@ -42,8 +39,12 @@
 	<DataTemplate x:Key="DefaultSidebarItemTemplate">
 		<local:SidebarItem
 			AutomationProperties.AutomationId="{Binding Text, Mode=OneWay}"
+			Decorator="{Binding ItemDecorator, Mode=OneWay}"
+			Icon="{Binding IconSource, Mode=OneWay}"
 			IsExpanded="{Binding IsExpanded, Mode=TwoWay}"
-			Item="{Binding}" />
+			Item="{Binding}"
+			Text="{Binding Text, Mode=OneWay}"
+			TooltipText="{Binding ToolTip, Mode=TwoWay}" />
```

Comment:
There was an issue with display of Text and Tooltip.
Using `RelativeSource` instead of implicit one.

Reply:
What was the issue?

Reply:
Didn't update on changed because those dont implement that logic but dependency prop does.